src/form-update.rst

```diff
@@ -98,6 +98,12 @@ These changes can be made as:
 
   "Example_form", "example_id", "English", "2017120701"
 
+.. note::
+
+  You need to use the same filename when generating newer versions of your form since the XLSForm generator pays attention to the filename. If you use different filenames, you would need to delete the previous version to upload the newer version of form.
```

Comment:
This is hard thing to communicate!

I'd say, "You need to use the same filename when generating newer versions of your form since the XLSForm converter uses that filename in creating the form structure the server uses to help determine uniqueness. If you use different filenames, you would need to delete the old version of the form to upload the newer version."